### ISSUE TITLE
Issue template: "Proxy used" is confusing

### ISSUE THREAD

**Issue by rugk** [keepassxreboot/keepassxc-browser#78]
I think it is very suboptimal of you to ask in the issue template, whether the "proxy [is] used". Besides the fact that you forgot to check your capslock key in yes/no answers (:wink:, srsly, this capslock is not necessary), there is a bigger problem here:

I assume you mean the KeePassXC proxy- application. However, I also did it wrong first and actually thought you mean the network proxy. Especially as we are in a browser context here, this is an error many users likely do. :laughing:

**Comment by varjolintu** [comment]
True.. this needs some clarification. The documentation needs a big update anyway.
